admin/helpers/pupil-presenter.js

```diff
@@ -10,7 +10,8 @@ const pupilPresenter = {}
  */
 pupilPresenter.getPupilExampleYear = () => {
   const deductionYears = 8
-  return moment.utc().format('YYYY') - deductionYears
+  const currentYear = parseInt(moment.utc().format('YYYY'), 10)
```

Comment:
Possibly simpler, as it eliminates the need to convert from string to number:

```suggestion
  const currentYear = (new Date()).getFullYear() // returns number
```